### ISSUE TITLE
GUI testing on CI

### ISSUE THREAD

**Issue by emilk** [rerun-io/rerun#3736]
We want to test our GUI on the CI, at least with some simple regression tests.

**Comment by emilk** [comment]
For this to be really useful we need some egui support for automation, i.e. something like https://github.com/ocornut/imgui_test_engine

We likely also want a software renderer for screenshot regression tests

**Comment by emilk** [comment]
This is unblocked by:
* https://github.com/emilk/egui/pull/5166

**Comment by emilk** [comment]
Duplicate of https://github.com/rerun-io/rerun/issues/2255
